src/FluentMigrator.Runner.Snowflake/Generators/Snowflake/SnowflakeQuoter.cs

```diff
@@ -96,8 +96,9 @@ public override string FormatSystemMethods(SystemMethods value)
                 case SystemMethods.NewGuid:
                     return "UUID_STRING()";
                 case SystemMethods.CurrentDateTimeOffset:
-                case SystemMethods.CurrentDateTime:
                     return "CURRENT_TIMESTAMP()";
+                case SystemMethods.CurrentDateTime:
+                    return "SYSDATE()";
```

Comment:
Can you also update the following unit tests so that they pass:

1. SnowflakeColumnTests.CanCreateColumnWithSystemMethodAndDefaultSchema 
2. SnowflakeColumnTests.CanCreateColumnWithSystemMethodAndCustomSchema